visualization/vignettes/Reworking of Visualization Tutorial.R

```diff
@@ -0,0 +1,58 @@
+library(PEcAn.all)
```

Comment:
Ok, now I'm really confused -- this looks like a 3rd version of the visualization tutorial, but now in a completely different place. I suspect this file should be deleted.

Also, try to avoid spaces in filenames -- makes it hard to work with in linux/unix